mediaire_toolbox/transaction_db/transaction_db.py

```diff
@@ -104,14 +109,23 @@ def create_transaction(self, t: Transaction) -> int:
         try:
             t.task_state = TaskState.queued
             self.session.add(t)
+            # when we commit, we get the transaction ID
+            self.session.commit()
+            if user_id:
```

Comment:
should we allow an entry in the user_transaction table if the user does not exist?